minos/api_gateway/rest/service.py

```diff
@@ -25,7 +28,12 @@ def __init__(self, address: str, port: int, config: MinosConfig):
         super().__init__(address, port)
 
     async def create_application(self) -> web.Application:
-        app = web.Application()
+        middlewares = []
```

Comment:
When empty lists are created, it's preferable to directly call `list()`
```suggestion
        middlewares = list()
```

Reply:
Done!